NOTICE.md

```diff
@@ -1,541 +1,539 @@
->    jurisdiction, without reference to its conflict-of-law provisions. Nothing
->    in this Section shall prevent a party's ability to bring cross-claims or
->    counter-claims.
-> 
-> 9. Miscellaneous
-> 
->    This License represents the complete agreement concerning the subject
->    matter hereof. If any provision of this License is held to be
->    unenforceable, such provision shall be reformed only to the extent
->    necessary to make it enforceable. Any law or regulation which provides that
->    the language of a contract shall be construed against the drafter shall not
->    be used to construe this License against a Contributor.
-> 
-> 
-> 10. Versions of the License
-> 
-> 10.1. New Versions
-> 
->       Mozilla Foundation is the license steward. Except as provided in Section
->       10.3, no one other than the license steward has the right to modify or
->       publish new versions of this License. Each version will be given a
->       distinguishing version number.
-> 
-> 10.2. Effect of New Versions
-> 
->       You may distribute the Covered Software under the terms of the version
->       of the License under which You originally received the Covered Software,
->       or under the terms of any subsequent version published by the license
->       steward.
-> 
-> 10.3. Modified Versions
-> 
->       If you create software not governed by this License, and you want to
->       create a new license for such software, you may create and use a
->       modified version of this License if you rename the license and remove
->       any references to the name of the license steward (except to note that
->       such modified license differs from this License).
-> 
-> 10.4. Distributing Source Code Form that is Incompatible With Secondary
->       Licenses If You choose to distribute Source Code Form that is
->       Incompatible With Secondary Licenses under the terms of this version of
->       the License, the notice described in Exhibit B of this License must be
->       attached.
-> 
-> Exhibit A - Source Code Form License Notice
-> 
->       This Source Code Form is subject to the
->       terms of the Mozilla Public License, v.
->       2.0. If a copy of the MPL was not
->       distributed with this file, You can
->       obtain one at
->       http://mozilla.org/MPL/2.0/.
-> 
-> If it is not possible or desirable to put the notice in a particular file,
-> then You may include the notice in a location (such as a LICENSE file in a
-> relevant directory) where a recipient would be likely to look for such a
-> notice.
-> 
-> You may add additional accurate notices of copyright ownership.
-> 
-> Exhibit B - "Incompatible With Secondary Licenses" Notice
-> 
->       This Source Code Form is "Incompatible
->       With Secondary Licenses", as defined by
+# NOTICES
+
+This repository incorporates material as listed below or described in the code.
+
+## Ensure.That 9.2.0
```

Comment:
Do we still need to list all of these individual software licenses?

Reply:
Yes. This is just a file name change and an adjustment to the header per this documentation - https://docs.opensource.microsoft.com/using/required-notice-template/

Reply:
@rogordon01 @rbhaiya Based on our conversations yesterday, it appears that we no longer need this file. Do we want to shutdown this PR, archive a copy of the file and then remove it completely?

Reply:
The PR can be abandoned for now. I looked yesterday and all of the software listed here _seems_ to be pulled in as a reference. We can confirm with @dustinburson and @namalu  when they come back. At which point we can issue a new PR to delete the file.
The file will always be in a our Git history, so no need to manual archive it.